AI-and-Analytics/DevCloudREADME.md

```diff
@@ -0,0 +1,27 @@
+## Running Samples In DevCloud 
+### Run Samples in Jupyter Lab
+1.	Open [Intel DevCloud](https://software.intel.com/content/www/us/en/develop/tools/devcloud.html).
+2.	In the upper right corner, click Sign In.
+3.	Log in with your Intel account username and password.
+4.	Open Jupyter lab: https://jupyter.oneapi.devcloud.intel.com/
```

Comment:
Change the jupyter link to: https://jupyter.oneapi.devcloud.intel.com/user/u80183/lab/tree/Welcome.ipynb

Reply:
hi Ben. this is the link of your account. we couldn't use that link for all users.